docs/user_docs/overview/supported-addons.md

```diff
@@ -8,107 +8,217 @@ sidebar_label: Supported addons
 
 # Supported addons
 
-KubeBlocks, as a cloud-native data infrastructure based on Kubernetes, provides management and control for relational databases, NoSQL databases, vector databases, and stream computing systems; and these databases can be all added as addons. Besides databases, the KubeBlocks addon now also supports plugins for cloud environments and applications.
+KubeBlocks uses Addons to extend support for various database engines. 
+There are currently over 30 Addons available in the KubeBlocks repository.
+Which can be further categorized as follows sections.
```

Comment:
It seems that these two sentences can be combined into one sentence with an attributive clause.

Reply:
yeah, that reads better